repo/packages/S/spark/0/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "object",
+  "properties": {
+    "spark/port": {
```

Comment:
On the contrary, I like to see a predictable 1:1 correspondence between the command line flag name and the property name.  There's a description, which should serve to disambiguate it from other port-related things.  If anything, maybe the description could be improved a little bit.